src/Parser.php

```diff
@@ -380,6 +415,79 @@ public function getCurrentToken(): Token
         return $this->stream->getCurrent();
     }
 
+    public function getFunction(string $name, int $line): TwigFunction
+    {
+        if (!$function = $this->env->getFunction($name)) {
```

Comment:
Those methods need to be updated to preserve the fix done in https://github.com/twigphp/Twig/pull/4575

Reply:
Rebased